app/views/batch_mailer/update_to_rates.html.slim

```diff
@@ -0,0 +1,81 @@
+p.salutation= t "publisher_mailer.shared.salutation", name: @publisher.name
+
+p Thank you for using the Brave referral system. We have a pretty important change coming up and we wanted to give you advance notice.
+
+p The per user payout rate will be changing from $5 globally to variable rates depending upon the region of the user
+
+strong Why are we doing this?
+
+p
+  div When we launched the referral program, the goal was to discover which areas would we see the most growth and to test the viability of the referral program as a whole. Our level of zoom was pretty wide (Planet Earth) and the rate was $5 per user, globally.
+  div Since then, we have launched our Ads program and we want to shift our level of zoom to more regional efforts, with a specific focus on the areas where we can deliver ads.  This is a straightforward business decision.
+
+strong When will this occur?
+p
```

Comment:
Need line break before this line

Reply:
https://github.com/brave-intl/publishers/pull/2203/commits/e8caff78e06e67e264e363660e585f0d02fe56d3